Packs/SysAid/Integrations/SysAid/SysAid.py

```diff
@@ -916,6 +923,22 @@ def service_record_attach_file_command(client: Client, args: Dict[str, Any]) ->
     return command_results
 
 
+def service_record_get_file_command(client: Client, args: Dict[str, Any]):
+    sr_id = str(args.get('id'))
+    file_id = str(args.get('file_id'))
+    file_name = str(args.get('file_name'))
+
+    response = client.service_record_get_file_request(sr_id, file_id)
+
+    if response.status_code == 200:
+        file_data = response.content
+        return_results(fileResult(file_name, file_data))
+
+        attachment_list = []
+        attachment_list.append({"file_name": file_name, "data": file_data})
+        return_results(str(attachment_list))
```

Comment:
Why is this required?

Reply:
You are right, I added those lines later.
I added them because running the command "!sysaid-service-record-get-file file_id=FILEID file_name=FILENAME id=IDTICKET" the command returns the download of the file in the WarRoom and context, but if this command is run inside an automation nothing is downloaded.
I added these 3 lines so that I would receive a string in the automation that contains all the file_name and file_data and be able to run the "fileResult" command from within the automation as well.